src/wazuh_db/wdb_agent.c

```diff
@@ -1242,6 +1243,38 @@ int wdb_remove_group_from_belongs_db(const char *name, int *sock) {
     return result;
 }
 
+int wdb_reset_agents_connection(int *sock) {
+    int result = 0;
+    char wdboutput[WDBOUTPUT_SIZE] = "";
```

Comment:
As we discussed with @vikman90 , we should make use of dynamically allocated memory instead of stack buffers. In fact, all the other methods will be updated in the near future to work in this way.

Reply:
Done